packages/lib-async-states/package.json

```diff
@@ -30,6 +30,9 @@
     "snazzy": "~8.0.0",
     "standard": "~12.0.1"
   },
+  "engines": {
+    "node": ">=0.12.18"
```

Comment:
node 0.12?

Reply:
That's all it needs, this library is tiny enough that I could manually check exactly what's required on node.green